WebHostLib/static/assets/tutorial/zeldaOOT/zeldaOOT_en.md

```diff
@@ -0,0 +1,387 @@
+# Setup Guide for Ocarina of time Archipelago
+
+## Important
+
+As we are using Z5Client and Bizhawk, this guide is only applicable to Windows.
+
+## Required Software Requerido
+
+- [bizhawk+script+Z5Client](https://github.com/ArchipelagoMW/Z5Client/releases) We recommend download Z5Client-setup as it makes some steps automatic.
+
+## Install Emulator and client
+
+Download getBizhawk.ps1 from previous link. Place it on the folder where you want your emulator to be installed, right click on it and select "Run with PowerShell". This will download all the needed dependencies used by the emulator. Can take a while.
```

Comment:
```suggestion
Download getBizhawk.ps1 from previous link. Place it on the folder where you want your emulator to be installed, right click on it and select "Run with PowerShell". This will download all the needed dependencies used by the emulator. This can take a while.
```